### PR TITLE
Content: STAD day plans update

### LINKED ISSUE #1168: Assign an energiser to the day plans in ITP
Zoom School is suffering from nobody planning or taking ownership, and the energiser especially. It might help to assign something specific, at least for the first half.

Suggest

1. Popcorn show and tell to teach popcorning
2. Screenshare show and tell to check screensharing
3. Breakout and introduce each other to check breakout room
4. Slack mad libs to bring their attention to Slack

### LINKED ISSUE #1182: Either change or replace the CRUD workshop in Module 2 Sprint 3
**Which module(s) and week(s) does this change affect?**
Module(s): Structuing Data
Week(s): 3

**What is the work that needs to be done?**

[Tony pointed out](https://codeyourfuture.slack.com/archives/C012UUW69S8/p1730978041860639)

"CRUD 101 workshop in Sprint 3 of ITP under Structuring and testing data suggests using Express and Nodejs to implement CRUD functionality in a server but at this point in ITP, the trainees are only on client side javascript.  Is this the right workshop suggestion for this step or has this CRUD 101 workshop been put into the wrong step"

**Why is this work important to do?**

**Additional context**

**Who might need to know about this change?**

### PR DESCRIPTION
**Pull request by SallyMcGrath**
## What does this change?

- slowly fade onboarding "blockers" and "wrap"  into standups and retros
- assign workshops they have context for
- add structured activities to introduce coursework tasks

Fixes #1182
Fixes #1168 
### Common Content?

- standup block
- tweaked some energisers

### Org Content?

STAD | 123 | Day Plans 

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [ ] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

File: org-cyf-itp/content/structuring-data/sprints/1/day-plan/index.md [hunk 1]
```diff
@@ -5,35 +5,40 @@ emoji= '🧑🏾‍🤝‍🧑🏾'
...
-  [[blocks.nested.blocks]]
-    name="Javascript Test Your Understanding [Tech] (30 Mins)"
-    src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/js1-wk1-eval"
-    time="0"
+name="Morning break"
+src="blocks/morning-break"
+[[blocks]]
+name="Another Asking Questions Workshop"
+src="https://cyf-pd.netlify.app/blocks/asking-good-technical-questions"
+time=70
```
Comment (illicitonion) [inline org-cyf-itp/content/structuring-data/sprints/1/day-plan/index.md:24]: This workshop mentions prep, but we haven't assigned the prep in the prep or backlog?

**Review by illicitonion — APPROVED** [review]
LGTM, just one thing!

File: org-cyf-itp/content/structuring-data/sprints/2/day-plan/index.md [hunk 2]
```diff
@@ -5,35 +5,38 @@ emoji= '🧑🏾‍🤝‍🧑🏾'
...
+[[blocks]]
+name="Time Traveller"
+src="energisers/time-traveller"
+time=65
 [[blocks]]
 name="lunch"
 src="blocks/lunch"
 [[blocks]]
+name="Stand Up"
+src="https://github.com/CodeYourFuture/CYF-Workshops/tree/main/stand-up"
```
Comment (illicitonion) [inline org-cyf-itp/content/structuring-data/sprints/2/day-plan/index.md:31]: I tweaked this a bit in https://github.com/CodeYourFuture/CYF-Workshops/pull/284

File: org-cyf-itp/content/structuring-data/sprints/1/day-plan/index.md (hunk 1 above)
Comment (SallyMcGrath) [reply org-cyf-itp/content/structuring-data/sprints/1/day-plan/index.md:24]: Combined into https://github.com/CodeYourFuture/Module-Structuring-and-Testing-Data/issues/11

**Pull request merged**